README.md

```diff
@@ -1,5 +1,7 @@
 **This is not an officially supported Google product.**
 
+This plugin supports Nullsoft Scriptable Install System (NSIS) binaries built using the NSIS builder available from https://nsis.sourceforge.io/
```

Comment:
My fault, I think we should call it an "extension" and not a plugin. Ghiidra also uses the term 'plugin' elsewhere.